### ISSUE TITLE
Unable to generate code in Visual Studio Code. SSL Error

### ISSUE THREAD

**Issue by Johnnycyan** [THUDM/CodeGeeX#50]
When trying to generate code I receive this error:

```
There was an error sending the request
Error: write EPROTO 32950664:error:100000f7:SSL routines:OPENSSL_internal:WRONG_VERSION_NUMBER:../../third_party/boringssl/src/ssl/tls_record.cc:242:
```

**Comment by Stanislas0** [comment]
Hi! Do you have internet access? Or is there any firewall enabled? This might cause unexpected connection errors.

**Comment by Johnnycyan** [comment]
Yes I have internet access. I previously used Github Copilot so I know that Visual Studio Code has access to the internet. Is there anything special I need to add to my firewall?

> Hi! Do you have internet access? Or is there any firewall enabled? This might cause unexpected connection errors.

**Comment by Stanislas0** [comment]
Could you please provide the version of your CodeGeeX and VS Code? Let us check.

**Comment by Johnnycyan** [comment]
> Could you please provide the version of your CodeGeeX and VS Code? Let us check.

CodeGeeX version: v1.1.1
Visual Studio Code version:  1.74.3

**Comment by Johnnycyan** [comment]
Update: It appears Malwarebytes is blocking a connection to tianqui.aminer.cn. What is this website used for?

**Comment by Stanislas0** [comment]
> Update: It appears Malwarebytes is blocking a connection to tianqui.aminer.cn. What is this website used for?

Hi! tianqi.aminer.cn is the site for CodeGeeX API. So it's necessary for CodeGeeX to work.

**Comment by Johnnycyan** [comment]
Alright. Might wanna add to the FAQ that it gets blocked by Malwarebytes
